[pull] primary from flybywiresim:primary (#229)
Valastiri/fbw-userguide: docs/fbw-a32nx/a32nx-api/index.md

# FlyByWire A32NX API

This section is aimed at sim pilots who want to use external hardware or software to connect to the FlyByWire A32NX to read values and control the aircraft.

## General

Many sim pilots want to use dedicated hardware or specific software to control their aircraft which goes beyond just the normal flight stick and maybe controller for thrust.

> The hardware alone is not sufficient to control aircraft in Microsoft Flight Simulator (or any sim). A software is required to tell the sim what the different levers (axis) and buttons shall do.

Reviewer: ```suggestion
The hardware alone is not sufficient to control aircraft in Microsoft Flight Simulator (or any sim). Software is required to tell the sim what the different levers (axis) and buttons shall do.
```

Most hardware (or software) vendors therefore ship their products with a driver for their hardware which translates hardware input into software commands which are then sent to the simulator.

> The problem with this approach is that for this to work all aircraft need to actually use the same software commands (API) for this to work on any aircraft. For Microsoft Flight Simulator this has been achieve by most default aircraft delivered with MSFS itself.
> 
> Unfortunately the API used (Simconnect, MSFS API) is not able to handle more complex aircraft and in addition there are other limitations which would go beyond this guide to explain.

Reviewer: ```suggestion
The problem with this approach is that all aircraft need to actually use the same software commands (API) for this to work on. For Microsoft Flight Simulator this has been achieved by most default aircraft delivered when MSFS launched. Unfortunately, the API used (Simconnect, MSFS API) is not able to handle more complex aircraft and in addition, there are other limitations that would go beyond this guide to explain.
```

combine so it flows better + general grammar
